external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -29,45 +30,36 @@ if(${ENABLE_PCMSolver})
                        -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
                        -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DPYMOD_INSTALL_LIBDIR=${PYMOD_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
                        -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DENABLE_psi4=ON
```

Comment:
I am not sure I want to introduce host-dependent flags. See also [here](https://github.com/PCMSolver/pcmsolver/commit/78dc78d20dd7f178bcd916ac450f865c48d08368#diff-a94fc1ec48247bd2a42eed10a9f07894)